### ISSUE TITLE
Time manager

### ISSUE THREAD

**Issue by nguyencongminh090** [gomocalc/gomocalc.github.io#10]
I'm curious about time manager work. Is it same with Piskvork?
Can you show the time clock?
Thanks!

**Comment by dhbloo** [comment]
Yes, the time manager works like that in piskvork. The AI side is given a limited time per turn and time per match.
Time clock is indeed a important feature to have. I will add it in the future.
